app-modules/service-management/src/Filament/Resources/ServiceRequestResource/Pages/ViewServiceRequest.php

```diff
@@ -216,13 +229,6 @@ protected function getHeaderActions(): array
     {
         return [
             EditAction::make(),
-            Action::make('locked_service_request')
-                ->icon('heroicon-o-lock-closed')
-                ->color('gray')
-                ->tooltip('This service request is locked as status is closed.')
-                ->disabled()
-                ->visible(fn (ServiceRequest $record) => fn (ServiceRequest $record) => $record->status?->classification === SystemServiceRequestClassification::Closed ? false : true)
-                ->iconButton(),
```

Comment:
Why was this action removed? It might have been a mistake.

Reply:
Done